src/testModel.ts

```diff
@@ -57,7 +58,7 @@ export class TestModel {
   private _envProvider: () => NodeJS.ProcessEnv;
   isEnabled = false;
   readonly tag: vscodeTypes.TestTag;
-  private _configError: reporterTypes.TestError | undefined;
+  private _errors = new MultiMap<string, reporterTypes.TestError>();
```

Comment:
To make it obvious what the key is:

```suggestion
  private _errorsByFile = new MultiMap<string, reporterTypes.TestError>();
```